### ISSUE TITLE
NMF solution: error with file being used by another process

### ISSUE THREAD

**Issue by agarciadom** [bluezio/ttc2016-live#11]
Note to @georghinkel: I understand that families2persons and class2rdb should be supported (since ForEach was added yesterday), but both are failing on my machine with Windows 10 after the code generation + compilation phase. Windows complains that the target model file is already being used by another process, I'm afraid.

Is it a quick bug fix you could do before the end of the first coffee break?

**Comment by georghinkel** [comment]
Yes, I also noticed that, though it does not seem to happen everytime. However, actually the solution works, the exception is thrown after the result has been serialized. Anyway, I will have a look at it, as it is clearly undesired behavior.

**Comment by agarciadom** [comment]
Thanks! I've checked that the actual files are generated correctly, so it's just a user experience issue :-). I've recorded the execution times.

By the way, for C2D I had to use a stopwatch, as it seems the main tool runs an executable and then closes without waiting for it to complete and reporting the full transformation + serialization time.

**Comment by georghinkel** [comment]
Yes, because I simply called the wrong method, the solution was not waiting for the transformation to complete but to another transformation to start. With the updated fix, it now measures also the time until the transformation completes. So also the original measurements for Families2Classes may be wrong.

**Comment by georghinkel** [comment]
Well, not just may, they actually have been wrong. Also the Families2Persons example yields execution times in the magnitude of seconds, at least for the large input models.
